packages/create-nx-workspace/bin/create-nx-workspace.ts

```diff
@@ -96,6 +98,10 @@ const presetOptions: { name: Preset; message: string }[] = [
     message:
       'react-native      [a monorepo with a single React Native application]',
   },
+  {
+    name: Preset.NodeServer,
+    message: 'node              [a standalone repo with a single Node Server]',
```

Comment:
Let's add `e.g. Express` since it's the most popular choice and we can be make obvious which selection to pick.